packages/zoe/src/internal-types.js

```diff
@@ -113,19 +113,34 @@
  *            ) => AddSeatResult} addSeat
  */
 
+/**
+ * @typedef {Object} AdminNode
+ * The AdminNode is a very powerful object with respect to the vat in which a
+ * contract runs. Anyone who holds it can cause the vat to shut down.
+ * @property {(reason: string) => void} terminate - terminate the vat in which the contract is
+ * running. This bypasses notification of Zoe. zcf.shutdown('reason') is much
+ * more graceful.
+ * @property {() => Record} adminData - return some statistics reflecting the
+ * behavior of this vat.
+ * @property {() => Promise} done - return a promise that will be rejected if
+ * the vat is terminated. Usually this will only be called after zcf.shutdown()
+ * has been called.
+ */
+
```

Comment:
This is already defined in line 224, I think

Reply:
Thanks. I'm surprised eslint didn't warn me. I'll combine.